### ISSUE TITLE
Showing error when trying to see a creator's other works

### ISSUE THREAD

**Issue by AmirMRezazad** [Mods-Center/Miui_Theme_Mod#4]
When i try to tap a theme creator profile it shows "Network Error" so i Can't see a theme or follow any content creator profile to see other work from him/her

**Comment by vk8437698** [comment]
@AmirMRezazad Well the theme app is based on China version. We changed server which provides themes. However creator's account and respectively server data is still linked to china one. Hence you can only see the profile of theme creator's in china region

**Comment by AmirMRezazad** [comment]
> @AmirMRezazad Well the theme app is based on China version. We changed server which provides themes. However creator's account and respectively server data is still linked to china one. Hence you can only see the profile of theme creator's in china region

Thanks for quick response, so for alternative solution, can you make theme app clone or change it to install and use it beside main and original one?

**Comment by vk8437698** [comment]
@AmirMRezazad Yeah, although it'll require proper disabling of apk vertification. As xiaomi exclusive system app requires their own signatures on apk. And to use unsigned apk you need to disable apk vertification.

**Comment by AmirMRezazad** [comment]
> @AmirMRezazad Yeah, although it'll require proper disabling of apk vertification. As xiaomi exclusive system app requires their own signatures on apk. And to use unsigned apk you need to disable apk vertification.

I'm not so good with these expertise  sir can you please show me a tutorial to disable it or it's up to a app dev to do this?

**Comment by AmirMRezazad** [comment]
Sorry i also check now and realized that can't add any widget from this modded theme app 
![Screenshot_2023-04-11-06-16-24-517_com miui personalassistant](https://user-images.githubusercontent.com/109852340/231042700-4df7cfd3-a767-493e-8cdd-aea15f172161.jpg)
![Screenshot_2023-04-11-06-16-45-946_com miui personalassistant](https://user-images.githubusercontent.com/109852340/231042712-d889fb88-34e8-49f8-a64c-73b9341b68ab.jpg)

**Comment by vk8437698** [comment]
@AmirMRezazad You need to use modded launcher and app vault as well from channel. You might not be able to find modded app vault in GitHub due to some unusual reasons. You can find it on telegram channel created by us

**Comment by Legion75** [comment]
> @AmirMRezazad You need to use modded launcher and app vault as well from channel. You might not be able to find modded app vault in GitHub due to some unusual reasons. You can find it on telegram channel created by us

the telegrama link from the github its broken, can u send a new one?
